_layouts/redirect.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  {% assign theLastVersion = site.data.versions.versions.last %}
+  {% for version in site.data.versions.versions %}
+    {% assign formatedVersion = version | replace: ".", "" %} 
+    {% assign releaseDetails = site.data.releases[formatedVersion] %} 
+    {% if releaseDetails.series.displayed %}
+    {%- assign sortedReleaseDetails = releaseDetails | sort -%}
+    {%- assign lastIndex = sortedReleaseDetails.size | minus: 2 -%}
+    {% assign formattedLastVersion = sortedReleaseDetails[lastIndex][0] %}
+      {% assign latestRelease = releaseDetails[formattedLastVersion] %}
+        {% if latestRelease.stable %}
+          {% assign latestStableVersion = version %}
+        {% endif %}
+    {% endif %} 
+  {% endfor%}
+  <head>
```

Comment:
Can we apply the proper style to this page? Currently, it shows up completely unstyled when encountering a redirect.

Reply:
Added the basic layout to redirect page. https://github.com/debezium/debezium.github.io/pull/591